<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="SearchB">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>60</y>
     <width>161</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="textE">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>60</y>
     <width>211</width>
     <height>341</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="CloseB">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>365</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>After Serch, Write the name of the slave</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="masterLE">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="OpenB">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>100</y>
     <width>141</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
